pkg/action/hooks.go

```diff
@@ -158,3 +177,62 @@ func hookHasDeletePolicy(h *release.Hook, policy release.HookDeletePolicy) bool
 	}
 	return false
 }
+
+// outputLogsByPolicy outputs a pods logs if the hook policy instructs it to
+func (cfg *Configuration) outputLogsByPolicy(h *release.Hook, releaseNamespace string, policy release.HookOutputLogPolicy) error {
+	if hookHasOutputLogPolicy(h, policy) {
+		namespace, err := cfg.deriveNamespace(h, releaseNamespace)
+		if err != nil {
+			return err
+		}
+		switch h.Kind {
+		case "Job":
+			return cfg.outputContainerLogsForListOptions(namespace, metav1.ListOptions{LabelSelector: fmt.Sprintf("job-name=%s", h.Name)})
```

Comment:
Unless I am mistaken, this is requiring jobs to have a `job-name=<hook name>` label for output to work

Reply:
I had to dig a little, but this is correct as-is. The release.Hook.Name is the Helm release name.